### PR TITLE
[CAP-20] Group Voting page and Linking Groups to BE

### PR DESCRIPTION
**Pull request by TAsif**
# Ticket
https://flickpicker.atlassian.net/browse/CAP-20

# Description
Second half of CAP-20

### CODE REVIEW

File: src/client/src/pages/LoginPage.jsx [hunk 1]
```diff
@@ -74,38 +71,31 @@ const BACKEND_URI = process.env.REACT_APP_FRONTEND_URI || 'http://localhost:8080
...
+        //   path: '/',
+        //   expires,
+        // });
+        // setCookie('expiration_time', obj.expirationTime, {
+        //   path: '/',
+        //   expires,
+        // });
         navigate('/home');
       })
-      .then ((userCredential) => {
```
Comment (madhi-naga) [inline src/client/src/pages/LoginPage.jsx:97]: why do we remove this?

Comment (madhi-naga) [reply src/client/src/pages/LoginPage.jsx:97]: won't we need to refer back to the user data? or this already handled by the cookie?

**Review by madhi-naga — APPROVED** [review]

File: src/client/src/pages/LoginPage.jsx (hunk 1 above)
Comment (TAsif) [reply src/client/src/pages/LoginPage.jsx:97]: It's in the cookies! This was only setting it for the current page which doesn't do anything for us, if you want to keep it persistent you have to either update the store or the cookies, both of which are already done

Comment (madhi-naga) [reply src/client/src/pages/LoginPage.jsx:97]: sweet, lgtm

**Review by ali-tab — APPROVED** [review]

**Pull request merged**